packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -52,6 +53,23 @@ export function SquadMemberModal({
     onMenuClick(e);
   };
 
+  const members = useMemo(
```

Comment:
Should we not rather do this on the backend?
Seems to be more memory efficient actually.

Reply:
Good point. We can save resources if we instead use the client's processing power (FE) for simple list manipulation than in BE as sorting is expensive in DB query. But definitely an option. WDYT? @idoshamun

Reply:
@rebelchris since the sorting won't probably be too heavy in the backend as we are at closed beta, I will open up a PR in the backend for that and close this one.

Reply:
I aggree with @rebelchris   this should be done in the be

Reply:
Yap agree. It's backend responsibility. Especially when paging involved (not now but probably in the future)